test/run_tests.bat

```diff
@@ -3,7 +3,8 @@ set MYDIR=%cd%\..
 set PYTHONPATH=%MYDIR%
 if [%1]==[] (
   SET date=%DATE%
-  python "%MYDIR%\scapy\tools\UTscapy.py" -t regression.uts -f html -o scapy_regression_test_%date:~6,4%_%date:~3,2%_%date:~0,2%.html
+  python "%MYDIR%\scapy\tools\UTscapy.py" -t regression.uts -K automaton -f html -o scapy_regression_test_%date:~6,4%_%date:~3,2%_%date:~0,2%.html

```

Comment:
Well this is a problem we should consider, i do not understand why it does not currently work... I'll check that a bit...